### PR TITLE
Implement calendar view task support

### PR DESCRIPTION
**Pull request by turtle96**
@Mamtha3005 @ZuoZhuolin @marcusngwj 
Note: need to recompile gradle/re-import gradle as the jfxtras library has been updated
should show event tasks only
image

Please test on your comp for blackouts, been trying to fix it
image
triggered sometimes by "hist", then some other command add, edit, done etc

### CODE REVIEW

**Comment by marcusngwenjian** [comment]
@turtle96 
Kindly fix merge conflict

**Comment by turtle96** [comment]
@marcusngwj fixed

**Comment by ZuoZhuolin** [comment]
image

Is it possible to remove this editing feature in the calendar? Because when users edit the date in the calendar, It will not sync with the left panel.

File: src/main/java/seedu/taskell/MainApp.java [hunk 1]
```diff
@@ -71,6 +72,7 @@ public void init() throws Exception {
         ViewHistoryCommand.getInstance().setData(model);
         SaveStorageLocationCommand.setConfig(config);
         SaveStorageLocationCommand.setStorage(storage);
+        CalendarView.setData(model);
     }
 
     private String getApplicationParameter(String parameterName){
```
Comment (marcusngwenjian) [inline src/main/java/seedu/taskell/MainApp.java:79]: whitespace should be between method signature and open braces

**Comment by Mamtha3005** [comment]
Documentation Reviewed

**Pull request merged**